manifests/examples/keyfactors/config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "caName": "<REPLACE_ME>",
+    "authToken": "<REPLACE_ME>",
+    "caTemplate": "<REPLACE_ME>",
+    "appKey": "<REPLACE_ME>",
+    "metadata": {
+        "Cluster": "Cluster",
+        "Service": "Service",
+        "PodName": "PodName",
```

Comment:
So these are like aliases or something? This could probably use a comment or something explaining how to set this. json doesn't have comments unfortunately so maybe a readme?

Reply:
Yes. This is all information about the Keyfactor environment. One platform feature allows metadata to be associated with a certificate. These are items that we hear from customers that they would like to track for their Istio certs, but we want to allow the flexibility for the metadata tags in Keyfactor to vary from customer to customer. @hovonako let's make sure this gets documented.